dll/win32/shell32/wine/PolicyData.h

```diff
@@ -0,0 +1,232 @@
+DEFINE_POLICY( REST_NOUPDATEWINDOWS,             Explorer,       NoWindowsUpdate                 )
+DEFINE_POLICY( REST_NOCHANGESTARMENU,            Explorer,       NoChangeStartMenu               )
+DEFINE_POLICY( REST_NOFOLDEROPTIONS,             Explorer,       NoFolderOptions                 )
+DEFINE_POLICY( REST_NOCSC,                       Explorer,       NoSyncAll                       )
+DEFINE_POLICY( REST_HASFINDCOMPUTERS,            Explorer,       FindComputers                   )
+DEFINE_POLICY( REST_RUNDLGMEMCHECKBOX,           Explorer,       MemCheckBoxInRunDlg             )
+DEFINE_POLICY( REST_INTELLIMENUS,                Explorer,       IntelliMenus                    )
+DEFINE_POLICY( REST_SEPARATEDESKTOPPROCESS,      Explorer,       SeparateProcess                 )
+DEFINE_POLICY( REST_MaxRecentDocs,               Explorer,       MaxRecentDocs                   )
+DEFINE_POLICY( REST_NOCONTROLPANEL,              Explorer,       NoControlPanel                  )
+DEFINE_POLICY( REST_ENUMWORKGROUP,               Explorer,       EnumWorkgroup                   )
+DEFINE_POLICY( REST_ARP_ShowPostSetup,           Uninstall,      ShowPostSetup                   )
+DEFINE_POLICY( REST_ARP_NOARP,                   Uninstall,      NoAddRemovePrograms             )
+DEFINE_POLICY( REST_ARP_NOREMOVEPAGE,            Uninstall,      NoRemovePage                    )
+DEFINE_POLICY( REST_ARP_NOADDPAGE,               Uninstall,      NoAddPage                       )
+DEFINE_POLICY( REST_GREYMSIADS,                  Uninstall,      NoWindowsSetupPage              )
+DEFINE_POLICY( REST_NOCHANGEMAPPEDDRIVELABEL,    Explorer,       NoChangeMappedDriveLabel        )
+DEFINE_POLICY( REST_NOCHANGEMAPPEDDRIVECOMMENT,  Explorer,       NoChangeMappedDriveComment      )
+DEFINE_POLICY( REST_NONETWORKCONNECTIONS,        Explorer,       NoNetworkConnections            )
+DEFINE_POLICY( REST_FORCESTARTMENULOGOFF,        Explorer,       ForceStartMenuLogoff            )
+DEFINE_POLICY( REST_NOWEBVIEW,                   Explorer,       NoWebView                       )
+DEFINE_POLICY( REST_NOCUSTOMIZETHISFOLDER,       Explorer,       NoCustomizeThisFolder           )
+DEFINE_POLICY( REST_NOENCRYPTION,                Explorer,       NoEncryption                    )
+DEFINE_POLICY( REST_DONTSHOWSUPERHIDDEN,         Explorer,       DontShowSuperHidden             )
+DEFINE_POLICY( REST_NOSHELLSEARCHBUTTON,         Explorer,       NoShellSearchButton             )
+DEFINE_POLICY( REST_NOHARDWARETAB,               Explorer,       NoHardwareTab                   )
+DEFINE_POLICY( REST_NORUNASINSTALLPROMPT,        Explorer,       NoRunasInstallPrompt            )
+DEFINE_POLICY( REST_PROMPTRUNASINSTALLNETPATH,   Explorer,       PromptRunasInstallNetPath       )
+DEFINE_POLICY( REST_NOMANAGEMYCOMPUTERVERB,      Explorer,       NoManageMyComputerVerb          )
+#if (NTDDI_VERSION < NTDDI_VISTA)
```

Comment:
```suggestion
#if (NTDDI_VERSION < NTDDI_LONGHORN)
```
(was already in the pre-releases)